docs/reference/msbuild-targets.md

```diff
@@ -464,6 +465,33 @@ msbuild -t:restore -p:RestorePackagesConfig=true
 > [!NOTE]
 > `packages.config` restore is only available with `MSBuild 16.5+`, and not with `dotnet.exe`
 
+### Restoring with MSBuild static graph evaluation
+
+With MSBuild 16.6+, NuGet has added the capability to use static graph evaluation from the commandline.
+This is an experimental feature that signficantly decreases the msbuild overhead of restore for large repos.
+
+```cli
+msbuild -t:restore -p:RestoreUseStaticGraphEvaluation=true
+```
+
+Alternatively you can enable it by setting the property in a Directory.Build.Props.
+
+> [!NOTE]
+> As of Visual Studio 2019.x and NuGet 5.x, this feature is considered experimental and opt-in. Follow [NuGet/Home#9803](https://github.com/NuGet/Home/issues/9803) for details on when this feature will be enabled by default.
+
+Static graph restore changes the msbuild part of restore, the project reading and evaluation, but not the restore algorithm! The restore algorithm is the same across all NuGet tools (NuGet.exe, MSBuild.exe, dotnet.exe and Visual Studio).
+
+In very few scenarios, static graph restore may behave differently from current restore and certain declared PackageReferences or ProjectReferences might be missing.
+
+To ease your mind and determine whether static graph works correctly for your scenario, consider running:
```

Comment:
Makes sense, maybe we just simplify to say "If static graph restore behaves differently, you can run a regular restore after".

We don't expect users to have to use two restores with this feature? Just in the case of a missing reference right?

Reply:
I meant to suggest that they should do this only once when migrating.

Reply:
Played with the wording a bit. I called out that it should be a one time check.

Let me know what you think.

Reply:
When we say "migrating to static graph restore" that really just means that the user opted into the feature and should do this once to make sure everything is populated. I'm good with the wording as changed!